iree/compiler/Dialect/HAL/Transforms/PackConstantPoolStorage.cpp

```diff
@@ -77,7 +77,8 @@ class PackConstantPoolStoragePass
       OpBuilder builder(poolOp.getContext());
       builder.setInsertionPointAfter(splatValueOp);
       auto splatOp = builder.create<ConstantPoolSplatOp>(
-          splatValueOp.getLoc(), splatValueOp.getName(),
+          splatValueOp.getLoc(), /*sym_visibility=*/nullptr,
+          splatValueOp.getName(),
```

Comment:
More of these... https://github.com/google/iree/pull/6964#discussion_r702137011

Any suggestions?

Reply:
commented there

Reply:
Rebased and added builders for each, matching their current usage. Call sites are simpler now, but I'm not fully satisfied with the ergonomics (optional args/attrs and implicit conversions still have coverage gaps for code that _should_ just work and isn't meaningfully different, IMO).